src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -299,6 +344,17 @@ public static int getWorldRenderPass()
     }
 
     public static void drawScreen(Screen screen, PoseStack mStack, int mouseX, int mouseY, float partialTicks)
+    {
+        mStack.pushPose();
+        guiLayers.forEach(layer -> {
+            drawScreenInternal(layer, mStack, -1, -1, partialTicks);
```

Comment:
If it is intended to pass `-1` as the mouse coordinates to the method, it may be worth adding a comment to that effect (so others who look at this aren't confused as to why it is like so).